src/sentry/static/sentry/app/views/settings/components/forms/field/fieldErrorReason.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'react-emotion';
+
+import {slideInUp} from 'app/styles/animations';
+
+const FieldErrorReason = styled('div')`
+  color: ${p => p.theme.redDark};
+  position: absolute;
+  right: 2px;
+  margin-top: 6px;
+  background: #fff;
+  padding: 6px 8px;
+  font-weight: 600;
+  font-size: 12px;
+  border-radius: 3px;
```

Comment:
This isn't one of our normal border-radius sizes. Should it use theme?

Reply:
I'd prefer not to change design right now